web_client/js/imageViewerWidget/leaflet.js

```diff
@@ -38,7 +38,8 @@ girder.views.LeafletImageViewerWidget = girder.views.ImageViewerWidget.extend({
                 [90.0, 180.0]
             ],
             layers: [
-                L.tileLayer(this._getTileUrl('{z}', '{x}', '{y}'), {
+                L.tileLayer(
+                    this._getTileUrl('{z}', '{x}', '{y}') + '?edge=%23DDD', {
```

Comment:
Does `this._getTileUrl` make the guarantee that it won't include a query string in its returned URL?